### PR TITLE
ICDS Dashboard: Split out topojsons

### PR DESCRIPTION
**Pull request by czue**
Updates to https://github.com/dimagi/commcare-hq/pull/25429/

### CODE REVIEW

**Comment by czue** [comment]
Ok @Rohit25negi I [wrapped my head around this] and feel decent about it. Can you please review?

File: custom/icds_reports/utils/topojson_util/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+The following is a brain dump of todos to improve this setup:
+
+- Reliably run `split_topojsons` and confirm it produces the same results
+- Implement this functionality for districts and not just blocks
+- Don't return the whole state's data when just the single district's blocks are needed
+- Maybe load the data via a static view instead of having it pass through django so it
+  can be served by nginx (would require porting some logic to the front end)
+- Or else, do some caching on the data so we aren't hitting the filesystem with every request
+- Confirm there are no duplcate district names
```
Comment (calellowitz) [inline custom/icds_reports/utils/topojson_util/README.md]: Im pretty sure there are definitely duplicate district names in different states (i think it caused a bug in the past). I do not believe there are duplicates in the same state.

**Review by calellowitz — APPROVED** [review]
looks like a big improvement for sure. left one comment but i think it is fine

File: custom/icds_reports/utils/topojson_util/README.md (hunk 1 above)
Comment (czue) [reply custom/icds_reports/utils/topojson_util/README.md]: Ok, I'll figure out how to test for duplicates before releasing this beyond mobile / feature flag.

Comment (czue) [reply custom/icds_reports/utils/topojson_util/README.md]: https://github.com/dimagi/commcare-hq/pull/26474

**Pull request merged**